server/utils/emailer.helper.js

```diff
@@ -0,0 +1,42 @@
+import dotenv from 'dotenv';
+import nodemailer from 'nodemailer';
+
+dotenv.config();
+
+const { EMAIL_ADDRESS, EMAIL_PASSWORD } = process.env;
+
+const transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: EMAIL_ADDRESS,
+    pass: EMAIL_PASSWORD,
+  },
+});
+
+/**
+ * Send Email to recipient
+ * @param {Object} emailPayload - Information about Email to be sent
+ * @param {String} emailPayload.email - The Recipient Email
+ * @param {String} emailPayload.subject - The Email Subject
+ * @param {String} emailPayload.message - The message to be sent can include html tags
+ * @returns {Object} Response
+ */
+const sendMail = (emailPayload) => {
+  const {
+    email,
+    subject,
+    message,
+  } = emailPayload;
+
+
+  const mailOptions = {
+    from: EMAIL_ADDRESS,
+    to: email,
+    subject,
+    html: message,
+  };
+
+  return transporter.sendMail(mailOptions);
```

Comment:
Handle the case of an error occurring when sending the mail.

Reply:
Thank you @paradisekelechi . We are working on it.